radicle-cli/src/commands/patch/create.rs

```diff
@@ -34,19 +34,19 @@ pub fn handle_patch_message(
         .message()
         .ok_or(anyhow!("commit summary is not valid UTF-8; aborting"))?;
     let message = message.get(&format!("{commit_message}{PATCH_MSG}"));
+    let message = message.replace(PATCH_MSG.trim(), ""); // Delete help message.
```

Comment:
Nice, though we should eventually improve `message.get()` to handle this

Reply:
Agreed.